### PR TITLE
Added noUpdate run parameter

### PR DESCRIPTION
**Pull request by ghost**
Added new run parameter  "noUpdate" to just pull new image without updating containers.

### CODE REVIEW

**Comment by ghost** [comment]
Requested in https://github.com/v2tec/watchtower/issues/138

**Comment by stffabi** [comment]
Thanks so much for your contribution. I'm currently a little bit busy, but I'll try to take a look into it soon.

File: actions/update.go [hunk 1]
```diff
@@ -34,7 +34,7 @@ func containerFilter(names []string) container.Filter {
 // used to start those containers have been updated. If a change is detected in
 // any of the images, the associated containers are stopped and restarted with
 // the new image.
-func Update(client container.Client, names []string, cleanup bool, noRestart bool) error {
+func Update(client container.Client, names []string, cleanup bool, noRestart bool, noUpdate bool) error {
```
Comment (stffabi) [inline actions/update.go:37]: What would you think about adding a new action `MonitorOnly`? As a result we wouldn't need to add the if statement in the update action.

File: main.go [hunk 2]
```diff
@@ -76,6 +77,11 @@ func main() {
 			Usage:  "remove old images after updating",
 			EnvVar: "WATCHTOWER_CLEANUP",
 		},
+		cli.BoolFlag{
```
Comment (stffabi) [inline main.go:80]: I think it would be nice to add an urfave command `monitor-only` instead of a flag, what do you think?

**Review by stffabi — COMMENTED** [review]
Thanks for your contribution.

I have added some minor comments.

**Comment by onedr0p** [comment]
@stffabi I think your requests make sense @ghost any update?

Edit: He is gone, @stffabi any chance you can add those changes? This is a nice feature to just get notified of updates.

**Comment by ghost** [comment]
This would be a great feature to have.

**Comment by tigpt** [comment]
+1

**Comment by shad-lp** [comment]
+1

**Comment by simonaronsson** [comment]
Implemented in commit d29b6cf93eeda8cc502ebc3a1d3bdd03b1efcc32.
Closing PR.

**Comment by onedr0p** [comment]
Wow, thought this project was dead.

**Comment by simonaronsson** [comment]
It was! But things are up and running again! :)

**Comment by robert-alfaro** [comment]
Thank you for this feature and picking things up again!!

**Pull request closed without merging**